Formula/ats2-postiats.rb

```diff
@@ -1,8 +1,8 @@
 class Ats2Postiats < Formula
-  desc "ATS programming language implementation"
+  desc "ATS2: Unleashing the Potentials of Types and Templates"
```

Comment:
This description feels worth to me than before; I have no idea what the new one means.